assets/training/model_management/environments/foundation-model-inference/context/score.py

```diff
@@ -130,58 +581,51 @@ def allocate_processes(hostfile_path):
-                range(
-                    LOAD_BALANCING_PORT + i * TENSOR_PARALLEL + 1,
-                    LOAD_BALANCING_PORT + i * TENSOR_PARALLEL + 1 + TENSOR_PARALLEL,
-                )
-            ),
-            torch_dist_port=i + TORCH_DIST_PORT,
-            gpu_indices=gpu_indices,
-        )
+            tensor_parallel_ports=list(range(LOAD_BALANCING_PORT+i*TENSOR_PARALLEL+1, LOAD_BALANCING_PORT+i*TENSOR_PARALLEL+1+TENSOR_PARALLEL)),
+            torch_dist_port=i+TORCH_DIST_PORT,
+            gpu_indices=gpu_indices
+        ) 
         for i, (hostname, gpu_indices) in enumerate(replica_pool)
     ]
-    load_balancer_config = LoadBalancerConfig(
-        port=LOAD_BALANCING_PORT, replica_configs=replica_configs
-    )
+    load_balancer_config = LoadBalancerConfig(port=LOAD_BALANCING_PORT, replica_configs=replica_configs)
     return load_balancer_config
 
 
 load_balancer_config = generate_load_balancer_config()
-
+is_70b_model = "Llama-2-70b" in MODEL_NAME or "Llama-2-70b-chat" in MODEL_NAME
+replace_with_kernel_inject = not is_70b_model
 configs = {
-    "deployment_name": "llama-deployment",
-    "ds_config": None,
-    "ds_optimize": True,
-    "ds_zero": False,
-    "load_balancer_config": load_balancer_config,
-    "mii_configs": {
-        "checkpoint_dict": None,
-        "deploy_rank": load_balancer_config.replica_configs[0].gpu_indices,
-        "dtype": torch.float16,
-        "enable_cuda_graph": False,
-        "enable_restful_api": False,
-        "hf_auth_token": None,
-        "load_with_sys_mem": True,
-        "max_tokens": MAX_TOKENS,
-        "meta_tensor": False,
-        "port_number": LOAD_BALANCING_PORT,
-        "profile_model_time": False,
-        "replace_with_kernel_inject": True,
-        "replica_num": REPLICA_NUM,
-        "skip_model_check": True,
-        "tensor_parallel": TENSOR_PARALLEL,
-        "torch_dist_port": TORCH_DIST_PORT,
-        "trust_remote_code": False,
+    'deployment_name': 'llama-deployment',
+    'ds_config': None,
+    'ds_optimize': True,
+    'ds_zero': False,
+    'load_balancer_config': load_balancer_config,
+    'mii_configs': {
+        'checkpoint_dict': None,
+        'deploy_rank': load_balancer_config.replica_configs[0].gpu_indices,
+        'dtype': torch.float16,
+        'enable_cuda_graph': False,
+        'enable_restful_api': False,
+        'hf_auth_token': None,
+        'load_with_sys_mem': True,
+        'max_tokens': MAX_TOKENS,
+        'meta_tensor': False,
+        'port_number': LOAD_BALANCING_PORT,
+        'profile_model_time': False,
+        'replace_with_kernel_inject': replace_with_kernel_inject,
+        'replica_num': REPLICA_NUM,
+        'skip_model_check': True,
+        'tensor_parallel': TENSOR_PARALLEL,
+        'torch_dist_port': TORCH_DIST_PORT,
+        'trust_remote_code': False
     },
-    "model_name": MODEL_NAME,
-    "model_path": MODEL_PATH,
-    "task_name": "text-generation",
+    'model_name': MODEL_NAME,
```

Comment:
Confused how `MODEL_NAME = os.getenv("AZUREML_MODEL_DIR", "")` works.

is `AZUREML_MODEL_DIR` stroring name of the model?

Reply:
updated the MODEL_NAME to MODEL_DIR.